lib/rules/no-unregistered-components.js

```diff
@@ -44,8 +44,10 @@ module.exports = {
         'disallow using components that are not registered inside templates',
       categories: null,
       recommended: false,
-      url: 'https://eslint.vuejs.org/rules/no-unregistered-components.html'
+      url: 'https://eslint.vuejs.org/rules/no-unregistered-components.html',
+      replacedBy: ['no-undef-components']
```

Comment:
Looks like this `replacedBy` needs to be outside of the `doc` object:
https://eslint.org/docs/developer-guide/working-with-rules#rule-basics

Reply:
Thanks, good catch! I fixed the position in https://github.com/vuejs/eslint-plugin-vue/pull/1783.